src/libraries/System.IO.Compression/src/System/IO/Compression/DeflateZLib/DeflateStream.cs

```diff
@@ -233,12 +314,38 @@ public override int ReadByte()
             return _inflater.Inflate(out b) ? b : base.ReadByte();
         }
 
+        /// <summary>Reads a number of decompressed bytes into the specified byte array.</summary>
+        /// <param name="buffer">The array to store decompressed bytes.</param>
+        /// <param name="offset">The byte offset in <paramref name="buffer" /> at which the read bytes will be placed.</param>
+        /// <param name="count">The maximum number of decompressed bytes to read.</param>
+        /// <returns>The number of bytes that were read into the byte array.</returns>
+        /// <remarks></remarks>
+        /// <example>The following example shows how to compress and decompress bytes by using the <see cref="O:System.IO.Compression.DeflateStream.Read" /> and <see cref="O:System.IO.Compression.DeflateStream.Write" /> methods.
+        /// <format type="text/markdown"><![CDATA[
+        /// [!code-csharp[System.IO.Compression.DeflateStream#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.io.compression.deflatestream/cs/program3.cs#3)]
+        /// [!code-vb[System.IO.Compression.DeflateStream#3](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.io.compression.deflatestream/vb/program3.vb#3)]
+        /// ]]></format></example>
+        /// <exception cref="System.ArgumentNullException"><paramref name="buffer" /> is <see langword="null" />.</exception>
+        /// <exception cref="System.InvalidOperationException">The <see cref="System.IO.Compression.CompressionMode" /> value was <see langword="Compress" /> when the object was created.
+        /// -or-
+        /// The underlying stream does not support reading.</exception>
+        /// <exception cref="System.ArgumentOutOfRangeException"><paramref name="offset" /> or <paramref name="count" /> is less than zero.
+        /// -or-
+        /// <paramref name="buffer" /> length minus the index starting point is less than <paramref name="count" />.</exception>
```

Comment:
The argument `buffer` is called `array` in the docs... should docs be changed?

https://docs.microsoft.com/dotnet/api/system.io.compression.deflatestream.read?view=net-5.0#System_IO_Compression_DeflateStream_Read_System_Byte___System_Int32_System_Int32_

Reply:
It looks like it was renamed in October with https://github.com/dotnet/runtime/commit/0acbafe06130149028c353c8aade702d6777edd5#diff-a076647d6d5119e4e3b482835f098318b585fc7fd9cb16116c45395721c61c3d.

What release is that in? And do we need to document it as a breaking change?

Reply:
@gewarren right, it is a breaking change that is going to ship with 6.0; it seems that the introducing PR https://github.com/dotnet/runtime/pull/43834 is already tagged with the `breaking-change` and the `needs-breaking-change-doc-created` labels.